Monika After Story/game/script-holidays.rpy

```diff
@@ -6304,6 +6304,26 @@ init 20 python:
  Monika
 """
     )
+    
+    mas_poem_vday_5 = MASPoem(
+        poem_id="poem_f14_5",
+        category="f14",
+        prompt="Sweetest",
+        title=" My dearest [player],",
+        text="""\
+Chocolate covered cherries and a goodnight kiss
+Being with you, darling, is the sweetest bliss.
+Popsicles in summer, dancing in the rain
+From snuggling you, honey, I could never abstain.
+Your laughter is my sugar cube, your smile is my cream
+To share a coffee date with you, my love, would be a dream.
+Of all the candy, all the cake, of fruits and cookies too
+The sweetest thing in my whole life will always be you.
```

Comment:
Because Journal does weird things with chopping some letters off, we add a space for these to make sure they don't get cut
```suggestion
 Chocolate covered cherries and a goodnight kiss
 Being with you, darling, is the sweetest bliss.
 Popsicles in summer, dancing in the rain
 From snuggling you, honey, I could never abstain.
 Your laughter is my sugar cube, your smile is my cream
 To share a coffee date with you, my love, would be a dream.
 Of all the candy, all the cake, of fruits and cookies too
 The sweetest thing in my whole life will always be you.
```